### PR TITLE
[C-2994] Improve mobile image selection

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Drastically improves mobile image selection for profile-picture, cover-art and album-art

### How Has This Been Tested?

Tested on ios and android simulators

### Screenshots

https://github.com/AudiusProject/audius-client/assets/8230000/3e0dfd23-e9f4-4ccc-93ce-16e7ccad1579

### CODE REVIEW

**Comment by audius-infra** [comment]
It looks like there may be some changes to native mobile code, which requires triggering a full app release. Please follow the instructions here: https://www.notion.so/audiusproject/When-to-bump-app-version-2644a8f772364a4d91f44abcba44ce0b?pvs=4. cc @nicoback2 @sliptype

File: packages/mobile/package.json [hunk 1]
```diff
@@ -117,6 +117,7 @@
     "react-native-get-random-values": "1.8.0",
     "react-native-google-cast": "4.6.0",
     "react-native-haptic-feedback": "1.11.0",
+    "react-native-image-crop-picker": "^0.40.0",
```
Comment (sliptype) [inline packages/mobile/package.json:120]: 📍

**Review by sliptype — APPROVED** [review]
Looks amazing! Love all the clean up too

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-c-2994-profile-cover-images-rotating

**Comment by audius-infra** [comment]
It looks like there may be some changes to native mobile code, which requires triggering a full app release. Please follow the instructions here: https://www.notion.so/audiusproject/When-to-bump-app-version-2644a8f772364a4d91f44abcba44ce0b?pvs=4. cc @nicoback2 @sliptype

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-c-2994-profile-cover-images-rotating

**Comment by audius-infra** [comment]
It looks like there may be some changes to native mobile code, which requires triggering a full app release. Please follow the instructions here: https://www.notion.so/audiusproject/When-to-bump-app-version-2644a8f772364a4d91f44abcba44ce0b?pvs=4. cc @nicoback2 @sliptype

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-c-2994-profile-cover-images-rotating

**Pull request merged**